src/app/learn/reference/helpers/index.html

```diff
@@ -11,19 +11,14 @@
   </p>
 
   <ul>
-    <li>
-      <p>
-        <stache-code>import &#123; SkyAuthHttp &#125; from '@blackbaud/skyux-builder/runtime</stache-code> — In conjunction with <a routerLink="/learn/reference/configuration">the <stache-code>"auth": true</stache-code> configuration setting in the <stache-code>skyuxconfig.json</stache-code> file</a>, we provide a wrapper to the <stache-code>http</stache-code> utility class. Use <stache-code>SkyAuthHttp.get()</stache-code> to automatically add the necessary Authorization header containing the bearer token received from authentication. This helper is for internal Blackbaud use only.
-      </p>
-    </li>
     <li>
       <p>
         <stache-code>import &#123; SkyHostBrowser &#125; from '@skyux-sdk/e2e</stache-code> — Particularly useful when running <a routerLink="/learn/reference/cli-commands">the <stache-code>skyux e2e</stache-code> command in the SKY UX CLI</a>. Use <stache-code>SkyHostBrowser.get()</stache-code> to automatically add the SKY UX Host base URL, as well as the necessary configuration object to enable local development mode. For example, <stache-code>SkyHostBrowser.get(/my-page?my-qs=1');</stache-code>.
       </p>
     </li>
     <li>
       <p>
-        <stache-code>import &#123; SkyAuthClientHttpModule &#125; from '@skyux/http';</stache-code>  — Enables SKY UX SPAs to use the <stache-code>HttpClient</stache-code> Angular service instead of the deprecated <stache-code>Http</stache-code> service. Use the <stache-code>skyAuthHttpOptions()</stache-code> function in HTTP calls to capture Blackbaud URL params such as <stache-code>leid</stache-code> and <stache-code>envid</stache-code>. For more information, see <a routerLink="/learn/reference/helpers/SkyAuthHttpClientModule">the <stache-code>SkyAuthHttpClientModule</stache-code> reference</a>.
+        <stache-code>import &#123; SkyAuthClientHttpModule &#125; from '@skyux/http';</stache-code>  — Enables SKY UX SPAs to use the <stache-code>HttpClient</stache-code> Angular service instead of the deprecated <stache-code>Http</stache-code> service. Use the <stache-code>skyAuthHttpOptions()</stache-code> function in HTTP calls to capture Blackbaud URL params such as <stache-code>leid</stache-code> and <stache-code>envid</stache-code>. This helper is for internal Blackbaud use only. For Blackbaud developers, the Engineering System website describes how to <a href="https://host.nxt.blackbaud.com/engineering-system-docs/learn/spa/spa-authenticated-calls">use <stache-code>SkyAuthHttpClientModule</stache-code> to make authenticated calls from a SPA to a web service</a>.
```

Comment:
Do you mind using `docs.blackbaud.com` instead of `host.nxt.blackbaud.com`?

Reply:
Nice catch. Sorry about the copy/paste error. ... I fixed the URL.